src/hypergol/tensorflow_model_manager.py

```diff
@@ -57,16 +57,6 @@ def restore_model_weights(self):
         """ restores tensorflow model weights """
         self.model.load_weights(f'{self.restoreWeightsPath}/{self.model.get_name()}.h5')
 
-    def initialize(self):
```

Comment:
took this out, because I didn't want to initialise bp here and close it in run(). we can put it back in a refactoring